source/fe/fe_system.cc

```diff
@@ -1204,8 +1118,28 @@ compute_fill (const Mapping<dim,spacedim>                      &mapping,
                           fe_data.update_flags);
 
 
-  if (flags & (update_values | update_gradients))
+  if (flags & (update_values | update_gradients | update_hessians))
     {
+      if (fe_data.is_first_cell())
+        {
+          // Initialize the internal FE objects for the base elements
+          for (unsigned int base_no=0; base_no<this->n_base_elements(); ++base_no)
```

Comment:
Yes you're right, it would have made my life easier to work on a branch.

So right now I've changed branch master so that it's identical to upstream master, and I have a branch that includes the changes (and only the necessary ones so it doesn't override other PR's) that I've rebased at the head of master. How should I proceed next? Should I merge the branch with master and push to origin? Or should I push the branch directly to origin?